docs/Tutorials/Python.md

```diff
@@ -24,3 +24,41 @@ supports `pip`, for instance in a
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
 with the command `. BUILD_DIR/bin/LoadPython.sh`.
+
+## Running Jupyter within the Docker container
+
+[Jupyter lab](https://jupyterlab.readthedocs.io/) is installed in the Docker
+container. You can run it in the container and access it through a browser on
+the host for a convenient way to work with the SpECTRE Python bindings. To do
+so, make sure you have exposed a port when running the Docker container, e.g.
+by appending the option `-p 8000:8000` to the `docker run` command (see
+\ref installation). Inside the docker container, it can be convenient to
+use `disown` or to `apt-get install screen` and use `screen` to obtain a shell
```

Comment:
Suggestion: "... use `disown` or `screen` (install this inside the container with `apt-get install screen`) to obtain..."

Reply:
`disown` doesn't give you a new shell, so that sounds misleading I think?

Reply:
That's a fair point, so let's forget this suggestion.